### PR TITLE
Prefix path in 4.0 version

### PR DESCRIPTION
**Pull request by wallacemaxters**

### CODE REVIEW

File: src/OVHSwiftAdapter.php [hunk 1]
```diff
@@ -83,8 +84,14 @@ public function getTemporaryUrl($path, $expiration, $options = []): string
     {
         $this->checkParams();
 
+        $path = $this->applyPathPrefix($path);
+
         // expiry is relative to current time
-        $expiresAt = $expiration instanceof Carbon ? $expiration->timestamp : (int) (time() + 60 * 60);
+        $expiresAt = $expiration;
```
Comment (iksaku) [inline src/OVHSwiftAdapter.php:90]: You’re breaking the timestamp implementation. Make sure to properly apply time arithmetics, as they’re needed for OVH’s timestamp validation.

**Review by iksaku — CHANGES_REQUESTED** [review]

**Comment by sausin** [comment]
Happy to merge when all issues are resolved

**Comment by sausin** [comment]
Any update on this?

**Comment by wallacemaxters** [comment]
Hello!

i will fix it

**Comment by sausin** [comment]
Thanks @wallacemaxters

**Pull request merged**